ios/MOSIPResidentApp.xcodeproj/project.pbxproj

```diff
@@ -321,7 +321,7 @@
 					"-ObjC",
 					"-lc++",
 				);
-				PRODUCT_BUNDLE_IDENTIFIER = io.mosip.inji.mobileid;
```

Comment:
We have changed the PRODUCT_BUNDLE_IDENTIFIER from io.mosip.inji.mobileid to io.mosip.inji.mobileid11, is it intentional, or we can revert this.